{"errorMessage":"Lambda process returned with error. Result: {\"errorType\":\"Runtime.ImportModuleError\",\"errorMessage\":\"Error: Cannot find module 'dotenv'\\nRequire stack:\\n- /var/task/apis/apiHandler.js\\n- /var/runtime/UserFunction.js\\n- /var/runtime/index.js\"}. Output:\n2021-07-30T14:30:09.454Z\tundefined\tERROR\tUncaught Exception \t{\"errorType\":\"Runtime.ImportModuleError\",\"errorMessage\":\"Error: Cannot find module 'dotenv'\\nRequire stack:\\n- /var/task/apis/apiHandler.js\\n- /var/runtime/UserFunction.js\\n- /var/runtime/index.js\",\"stack\":[\"Runtime.ImportModuleError: Error: Cannot find module 'dotenv'\",\"Require stack:\",\"- /var/task/apis/apiHandler.js\",\"- /var/runtime/UserFunction.js\",\"- /var/runtime/index.js\",\"    at _loadUserApp (/var/runtime/UserFunction.js:100:13)\",\"    at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)\",\"    at Object.<anonymous> (/var/runtime/index.js:43:30)\",\"    at Module._compile (internal/modules/cjs/loader.js:1063:30)\",\"    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)\",\"    at Module.load (internal/modules/cjs/loader.js:928:32)\",\"    at Function.Module._load (internal/modules/cjs/loader.js:769:14)\",\"    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)\",\"    at internal/main/run_main_module.js:17:47\"]}\n\u001b[32mSTART RequestId: 2e689c66-7734-1256-e7cb-0cc15296f9a0 Version: $LATEST\u001b[0m\n\u001b[32mEND RequestId: 2e689c66-7734-1256-e7cb-0cc15296f9a0\u001b[0m\n\u001b[32mREPORT RequestId: 2e689c66-7734-1256-e7cb-0cc15296f9a0\tInit Duration: 91.92 ms\tDuration: 0.93 ms\tBilled Duration: 100 ms\tMemory Size: 1536 MB\tMax Memory Used: 50 MB\t\u001b[0m","errorType":"InvocationException","stackTrace":["  File \"/opt/code/localstack/localstack/services/awslambda/lambda_api.py\", line 752, in run_lambda\n    lock_discriminator=lock_discriminator,\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 273, in execute\n    return do_execute()\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 263, in do_execute\n    return _run(func_arn=func_arn)\n","  File \"/opt/code/localstack/localstack/utils/cloudwatch/cloudwatch_util.py\", line 157, in wrapped\n    raise e\n","  File \"/opt/code/localstack/localstack/utils/cloudwatch/cloudwatch_util.py\", line 153, in wrapped\n    result = func(*args, **kwargs)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 250, in _run\n    raise e\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 236, in _run\n    result = self._execute(func_arn, func_details, event, context, version)\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 489, in _execute\n    event=stdin, env_vars=environment, func_details=func_details, docker_flags=docker_flags\n","  File \"/opt/code/localstack/localstack/services/awslambda/lambda_executors.py\", line 391, in run_lambda_executor\n    ) from error\n"]}
